application/view/omeka/admin/item-set/edit.phtml

```diff
@@ -4,8 +4,6 @@ $escape = $this->plugin('escapeHtml');
 $escapeAttr = $this->plugin('escapeHtmlAttr');
 $this->headScript()->appendFile($this->assetUrl('js/resource-form.js', 'Omeka'));
```

Comment:
Similarly, it seems that this line is now unnecessary here, since the script is included by the shared partial.